istio-pilot-agent-1.19.yaml

```diff
@@ -7,7 +7,7 @@ package:
     - license: Apache-2.0
   dependencies:
     provides:
-      - istio-pilot-agent=${{package.full-version}}
+      - istio-pilot-agent=1.19
```

Comment:
approved, but cc @vaikas since I also thought `${{package.full-version}}` is the canonical way of doing this now?

https://github.com/chainguard-dev/melange/blob/main/docs/BUILD-FILE.md#provides

Reply:
![image](https://github.com/wolfi-dev/os/assets/2442466/516ace2c-5587-44e8-9bc5-2b17ac99cb96)
 it is empty string, might need a melange update first.

Reply:
I believe this is because https://github.com/chainguard-dev/melange/pull/662 added the placeholder but we don't do expansion in `provides` / `dependencies` (we do in `pipeline`)